src/tool_cb_hdr.c

```diff
@@ -103,14 +116,26 @@ size_t tool_header_cb(char *ptr, size_t size, size_t nmemb, void *userdata)
   if(per->config->etag_save_file && etag_save->stream) {
     /* match only header that start with etag (case insensitive) */
     if(curl_strnequal(str, "etag:", 5)) {
-      char *etag_start = ptr + strlen("ETag:");
+      const char *etag_start = ptr + strlen("ETag:");
+      const char *expected_etag_end, *etag_end;
+      size_t etag_length;
 
       /* pass all spaces from start */
       while(*etag_start && ISSPACE(*etag_start))
         etag_start++;
 
+      /* We expect CRLF as EOL but when called from tests it may be CR or LF */
```

Comment:
This isn't a test thing. This is how HTTP works in the wild. And it can't be a CR-only.

Reply:
never saw this in a wild but guess you already had troubles with this :) The comment may be not so precise but RFC requires CRLF so for simplicity this comment may be fine. If you would like to change then please do because anyway you can write it better.

Reply:
I've told you this **many** times already in this PR.